### PR TITLE
Create zh trans host-aliases.md

### PR DESCRIPTION
**Pull request by jiazxjason**
Create zh trans host-aliases.md

### CODE REVIEW

File: content/zh/docs/reference/glossary/host-aliases.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+-->
+
+| 名称       | id          | 日期       | 链接                                                    | 简述                                            | aka  | 标签      |
+| ----------- | ----------- | ---------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ---- | --------- |
+| HostAliases | HostAliases | 2019-01-31 | /docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-core | HostAliases 是 IP 地址和要注入到 Pod 主机文件中的主机名之间的映射。 |      | 操作 |
+
+HostAliases 是 IP 地址和要注入到 Pod 主机文件中的主机名之间的映射。
+
+[HostAliases](https://github.com/kubernetes/website/blob/release-1.16/docs/reference/generated/kubernetes-api/v1.13/#hostalias-v1-corev) 是可选的主机名和IP地址列表，如果指定的话，将注入到Pod的主机文件中。 这仅对非主机网络 Pod 有效。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/host-aliases.md]: oh my god, the hyperlinks are terrible

Comment (jiazxjason) [reply content/zh/docs/reference/glossary/host-aliases.md]: the original link is broken and i can't find the correct one

Comment (tengqm) [reply content/zh/docs/reference/glossary/host-aliases.md]: suggestions:
- use relative path for in-site links, not absolute path;
- if there is no special requirement, we should  try pointing things to the current release, especially for API resources; You can grep for examples how current version can be inserted by hugo for you;
- pointing to 1.13 API in the 1.16 release is strange;
- if the English content is bad, please consider fix it as well.

Comment (zhangxiaoyu-zidif) [reply content/zh/docs/reference/glossary/host-aliases.md]: @joaovitor  you can use some parameter like {{current branch}} to replace the specified branch.

**Comment by markthink** [comment]
@jiazxjason 存在冲突，需要解决

**Pull request closed without merging**